### PR TITLE
Fix/orbit 6 issues

### PR DESCRIPTION
**Pull request by Rosomack**
Fixes for:

* https://github.com/orbit-mvi/orbit-mvi/issues/177
* https://github.com/orbit-mvi/orbit-mvi/issues/179
* https://github.com/orbit-mvi/orbit-mvi/issues/180

Blocking intents and other container host functions can be called via `containerHost.xxx()` directly.

I'm removing `invokeIntent` to keep things consistent cc @mattmook , see discussion at https://github.com/orbit-mvi/orbit-mvi/issues/180 . TL;DR is that this change will allow people not to expose `Job` from their `ContainerHost` if they don't need it in tests - I think this is the solution we missed originally.

### CODE REVIEW

**Review by TWiStErRob — APPROVED** [review]
These are non-breaking changes, if you remove a method that's definitely a v7.

File: orbit-test/src/commonMain/kotlin/org/orbitmvi/orbit/test/OrbitTestContext.kt [hunk 1]
```diff
@@ -31,6 +31,7 @@ public interface OrbitTestContext<STATE : Any, SIDE_EFFECT : Any, CONTAINER_HOST
     /**
      * Invoke an intent on the [ContainerHost] under test.
      */
+    @Deprecated("Use containerHost instead", ReplaceWith("action(containerHost)"))
```
Comment (TWiStErRob) [inline orbit-test/src/commonMain/kotlin/org/orbitmvi/orbit/test/OrbitTestContext.kt:34]: Does this work with IntelliJ quickfix? Is it the reason for the all the `this`s in `this.containerHost.x()`?

Comment (Rosomack) [reply orbit-test/src/commonMain/kotlin/org/orbitmvi/orbit/test/OrbitTestContext.kt:34]: Yes and yes!

**Comment by Rosomack** [comment]
> These are non-breaking changes, if you remove a method that's definitely a v7.

Yes, I would like to fix it quickly in a way that allows people to refactor (`ReplaceWith`)

`7.0.0` is already reserved for removing legacy style tests and the `invokeIntent` method will go with it

**Review by mattmook — APPROVED** [review]

**Pull request merged**